persistence/src/main/resources/com/zebrunner/reporting/persistence/dao/mappers/application/TestSessionMapper.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we use myBatis for the new entity instead of Spring Data?

Reply:
I can switch to spring-data-mybatis

Reply:
The point was about spring-data-jpa since integrations feature already uses it and this mapper as far as I can see is not connected with the others